Sources/StreamChat/ChatClient.swift

```diff
@@ -229,15 +235,6 @@ public class ChatClient {
     private let sessionHeaders: [String: String] = [
         "X-Stream-Client": SystemEnvironment.xStreamClientHeader
     ]
-    
-    /// The current connection id
-    @Atomic var connectionId: String?
-    
-    /// An array of requests waiting for the connection id
-    @Atomic private(set) var connectionIdWaiters: [String: (String?) -> Void] = [:]
```

Comment:
Great that we're getting rid of these - now they're in the correct place and no `@Atomic` is needed. 👍